packages/react-components/react-button/stories/Button/ButtonShape.stories.tsx

```diff
@@ -24,5 +24,29 @@ Shape.parameters = {
     description: {
       story: 'A button can be rounded, circular, or square.',
     },
+    source: {
+      code: `
+import * as React from 'react';
```

Comment:
copying code like this isn't preferable IMO, if usage or features change and the doc samples are not updated then users who copy+paste are left with incorrect usage.

At least currently it's pretty obvious that users don't see the full code file in the show code area but can click on export to codesandbox to see the real code.

Reply:
^ that's very valid point